packages/react-components/src/components/AppFrame/components/SideNavigationGroup/SideNavigationGroup.tsx

```diff
@@ -0,0 +1,132 @@
+import * as React from 'react';
+
+import { ChevronRight } from '@livechat/design-system-icons';
+import cx from 'clsx';
+
+import noop from '../../../../utils/noop';
+import { Icon } from '../../../Icon';
+import { Text } from '../../../Typography';
+import { SideNavigationItem } from '../SideNavigationItem/SideNavigationItem';
+
+import { ISideNavigationGroupProps } from './types';
+
+import styles from './SideNavigationGroup.module.scss';
+
+const baseClass = 'side-navigation-group';
+
+export const SideNavigationGroup: React.FC<ISideNavigationGroupProps> = ({
+  label,
+  rightNode,
+  className,
+  children,
+  isCollapsible,
+  onItemHover,
+  shouldOpenOnInit = false,
+  shouldOpenOnActive = false,
+}) => {
+  const [isOpen, setIsOpen] = React.useState(
+    !isCollapsible || shouldOpenOnInit
+  );
+  const [isSideMenuVisible, setIsSideMenuVisible] = React.useState(false);
```

Comment:
Why is this state named like this? How is this state related to side menu visibility?